web-client/src/app/components/header/header.component.html

```diff
@@ -0,0 +1,23 @@
+<ion-toolbar>
+  <ion-buttons slot="start">
+    <ion-button>
+      <ion-icon name="menu" src="icon-only"></ion-icon>
+    </ion-button>
+  </ion-buttons>
+  <ion-grid>
+    <ion-row class="ion-align-items-center">
+      <ion-col size="auto">
+        <img src="assets/img/Nautilus.svg" />
+      </ion-col>
+      <ion-col>
+        <ion-button
+          size="small"
+          expand="block"
+          shape="round"
+          color="portland-orange"
+          >DEPOSIT
+        </ion-button>
```

Comment:
I don't think we actually want a button here. The initial mock-ups have one, but see [this Figma thread](https://www.figma.com/file/zYVHu1EwCIDCbQp8rFscyA?node-id=39:322#94660054): let's talk about that before committing here?

Reply:
(I'm in favour of just leaving this out for now, rather than adding and having to remove it again later.)